### ISSUE TITLE
Only specifying some types of watermarks results in an error

### ISSUE THREAD

**Issue by nmalaguti** [karma-runner/karma-coverage#144]
This is blocked by issue #143.

Specifying only `statements` and `lines` results in an error from Istanbul.
```javascript
coverageReporter: {
  watermarks: {
    statements: [ 50, 75 ],
    lines: [ 50, 75 ]
  }
}
```

```
ERROR [coverage]: [TypeError: Cannot read property '1' of undefined]
TypeError: Cannot read property '1' of undefined
    at Object.module.exports.classFor (karma-coverage/node_modules/istanbul/lib/report/common/defaults.js:22:29)
    at tableRow (karma-coverage/node_modules/istanbul/lib/report/text.js:119:49)
    at walk (karma-coverage/node_modules/istanbul/lib/report/text.js:159:20)
    at karma-coverage/node_modules/istanbul/lib/report/text.js:162:9
    at Array.forEach (native)
    at walk (karma-coverage/node_modules/istanbul/lib/report/text.js:161:19)
    at TextReport.Report.mix.writeReport (karma-coverage/node_modules/istanbul/lib/report/text.js:200:9)
    at writeReport (karma-coverage/lib/reporter.js:69:16)
    at karma-coverage/lib/reporter.js:304:13
    at Array.forEach (native)
```

This appears to be because Istanbul expects the `watermarks` object to have all 4 types specified. I'm not sure if this is an issue with Istanbul or if karma-coverage should merge any specified `watermarks` with a set of defaults.

From https://github.com/gotwarlost/istanbul/blob/master/lib/report/common/defaults.js:
```javascript
{
  statements: [ 50, 80 ],
  lines: [ 50, 80 ],
  functions: [ 50, 80],
  branches: [ 50, 80 ]
}

**Comment by Dignifiedquire** [comment]
@nmalaguti can this be closed now that #145 is merged?

**Comment by nmalaguti** [comment]
Yes. Resolved by #145.
